src/library/sidebarmodel.cpp

```diff
@@ -234,13 +235,14 @@ QVariant SidebarModel::data(const QModelIndex& index, int role) const {
         if (tree_item) {
             if (role == Qt::DisplayRole) {
                 return tree_item->getLabel();
+
             } else if (role == Qt::ToolTipRole) {
                 // If it's the "Quick Links" node, display it's name
```

Comment:
The comment is obsolete now, please delete it.

Reply:
of course...